babel.config.js

```diff
@@ -10,7 +10,7 @@ let isTestFile = filename =>
 module.exports = api => {
   api.cache(true)
   return {
-    presets: ['babel-preset-emotion-dev'],
+    presets: ['babel-preset-emotion-dev', '@babel/preset-typescript'],
```

Comment:
isn't this conflicting with the flow plugin that is used at the same time?

Reply:
Probably but I haven't tried to build anything yet. Not sure how to coordinate that part exactly if do conflict. Just remove `flow` and assume we won't merge all of these branches till everything is converted? 

Added because `preconstruct` uses it.

Reply:
We should use https://babeljs.io/docs/en/options#overrides to make run @babel/preset-typescript on `.ts` and `.tsx` and `@babel/preset-flow` on `.js`. Ideally we should merge these incrementally.

Reply:
note that `@babel/preset-typescript` already has `overrides` in its implementation: https://github.com/babel/babel/blob/672a58660f0b15691c44582f1f3fdcdac0fa0d2f/packages/babel-preset-typescript/src/index.js#L35 . I'm not completely sure how well this composes with other stuff though.

Reply:
It doesn't appear to conflict. `build` works fine whether the preset is listed or not.